applications/crossbar/doc/conference.md

```diff
@@ -43,6 +43,12 @@ Key | Description | Type | Default | Required
 `play_name` | Do we need to announce new conference members? | `boolean()` | `false` | `false`
 `play_welcome` | Whether to play the welcome prompt | `boolean()` |   | `false`
 `profile` | The XML profile name used to configure the conference | `string()` |   | `false`
+
+
+
+#### Development not complete on the below ####
```

Comment:
This is not valid to do, as the scripts checking these schema tables are inserting the now missing fields from the main schema. This is what CircleCI is complaining about - the scripts added the rows back to the table above and now there are unstaged changes.

Instead, let's leave the rows in the table and add a `Note: ` below about the fields' status